mne/viz/evoked.py

```diff
@@ -875,3 +880,172 @@ def plot_snr_estimate(evoked, inv, show=True):
+    if exclude is not None:
+        evoked.drop_channels(exclude)
+
+    info = evoked.info
+    data_types = ['eeg', 'grad', 'mag', 'seeg']
+    ch_types = set(ch_type for ch_type in data_types if ch_type in evoked)
+
+    # if multiple sensor types: one plot per channel type, recursive call
+    if len(ch_types) > 1:
+        figs = list()
+        for t in ch_types:  # pick only the corresponding channel type
+            ev_ = evoked.pick_channels([info["ch_names"][idx]
+                                        for idx in range(info['nchan'])
+                                        if channel_type(info, idx) == t],
+                                       copy=True)
+            if len(set([channel_type(ev_.info, idx)
+                        for idx in range(ev_.info['nchan'])
+                        if channel_type(ev_.info, idx) in data_types])) > 1:
+                raise RuntimeError("Possibly infinite loop due to channel "
+                                   "selection problem. This should never "
+                                   "happen! Please check your channel types.")
+            figs.append(_joint_plot(ev_, times=times, title=title, show=show,
+                                    ts_args=ts_args, exclude=list(),
+                                    topomap_args=topomap_args))
+        return figs
+
+    fig = plt.figure()
+
+    # set up time points to show topomaps for
+    times = _process_times(evoked, times, few=True)
+
+    # butterfly/time series plot
+    ts_ax = fig.add_subplot(212)
+    ts_args_pass = dict((k, v) for k, v in ts_args.items()
+                        if k not in ["axes", "show", "colorbar"])
+    ts_args_pass["spatial_colors"] = (ts_args["spatial_colors"] if
+                                      "spatial_colors" in ts_args else True)
+    plot_evoked(evoked, axes=ts_ax, show=False, set_tight_layout=False,
+                **ts_args_pass)
+
+    # handle title
+    # we use a new axis for the title to handle scaling of plots
+    old_title = ts_ax.get_title()
+    ts_ax.set_title('')
+    if title is not None:
+        title_ax = plt.subplot(4, 3, 2)
+        title = ", ".join([title, old_title]) if len(title) > 0 else old_title
+        title_ax.text(.5, .5, title, transform=title_ax.transAxes,
+                      horizontalalignment='center',
+                      verticalalignment='center')
+        title_ax.axis('off')
+
+    # prepare axes for topomap
+    # slightly convoluted due to colorbar placement and for vertical alignment
+    ts = len(times) + 2
+    map_ax = [plt.subplot(4, ts, x + 2 + ts) for x in range(ts - 2)]
+    cbar_ax = plt.subplot(4, 3 * (ts + 1), 6 * (ts + 1))
+
+    # topomap
+    topomap_args_pass = dict((k, v) for k, v in topomap_args.items() if
+                             k not in ["times", "axes", "show", "colorbar"])
+    topomap_args_pass["outlines"] = (topomap_args["outlines"] if "outlines"
+                                     in topomap_args else 'skirt')
```

Comment:
You could do ``topomap_args_pass['outlines'] = topomap_args.get('outlines', 'skirt')``. And for consistency, use single quotes when possible.